sql/telemetry/feature_usage/query.sql

```diff
@@ -0,0 +1,83 @@
+SELECT
+  events_t.client_id AS clients_id,
+  events_t.submission_date AS submission_date,
```

Comment:
if `JOIN` is via `USING` instead of `ON`, then this can be:
```suggestion
  clients_id,
  submission_date,
```

Reply:
And it looks like we have a mispelling here: `clients_id`